src/API/Notes.php

```diff
@@ -327,15 +327,25 @@ public function get_requested_updates( $request ) {
 	}
 
 	/**
-	 * Undo delete all notes.
+	 * Batch update a set of notes.
 	 *
 	 * @param WP_REST_Request $request Request object.
 	 * @return WP_REST_Request|WP_Error
 	 */
-	public function undoremove_items( $request ) {
+	public function batch_update_items( $request ) {
```

Comment:
I realized this endpoint is already set up to batch update, which could be pretty useful.

Not sure if we need to add a deprecated method for `undoremove_items` if we go forward with this update.

Reply:
> Not sure if we need to add a deprecated method for undoremove_items if we go forward with this update.

I think that is the safe thing to do given prior feedback on matters like this.

Reply:
Added deprecated method and notice.